Doc/howto/annotations.rst

```diff
@@ -29,6 +29,39 @@ Annotations Best Practices
   in your code, please see the :mod:`typing` module.
 
 
+.. _annotations-howto-general:
+
+General rules
+=============
+
+When defining annotations, remember about the execution order. For example::
+
+   class User:
+      username: str = 'miss-islington'
+
+1. Value ``'miss-islington'`` will be assigned
+   to name ``username`` in ``User`` class scope
+2. Name ``str`` is evaluated in the annotated assignment statement
```

Comment:
This is incorrect in Python 3.14.

Reply:
Oh, yes, right! I guess we need to sections: one for 3.14+ and one for older versions.

How would you best describe the current approach?